theme/styles/buttons.scss

```diff
@@ -2,6 +2,10 @@ button {
   &:disabled {
     color: color('gray');
     cursor: not-allowed;
+
+    &:hover {
+      border: 1px solid color('transparent');
+    }
```

Comment:
Ensuring that the "Added to Cart" doesn't have the white border hover effect

Reply:
We can probably just give it a pointer events none, since it's not really a button once it's changed state